admin-console/src/main/webapp/src/components/EmailTemplates/CreateViewInstantTemplateButton.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+    Button
+} from '@patternfly/react-core';
+import React from 'react';
+
+interface InstantTemplateButtonProps {
+    hasTemplate: boolean;
+    createInstantTemplateButton: React.ReactNode
+
+}
+export const InstantTemplateButton: React.FunctionComponent<InstantTemplateButtonProps> = (props) => {
```

Comment:
Is this used anywhere?

Reply:
ah, no. i decided to just implement the button in the instantEmailCell. Will remove.